extensions/rest-raml/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@loopback/rest-raml",
+  "version": "0.1.0",
+  "description": "A lightweight extension to convert the generated OpenAPI specifications into RAML 1.0 and expose it via a binding and optionally via the REST server.",
+  "keywords": [
+    "loopback-extension",
+    "loopback",
+    "RAML"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "engines": {
+    "node": "^10.16 || 12 || 14 || 15"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "build:watch": "lb-tsc --watch",
+    "pretest": "npm run build",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "clean": "lb-clean dist *.tsbuildinfo .eslintcache"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strongloop/loopback-next.git",
+    "directory": "extensions/rest-raml"
+  },
+  "author": "IBM Corp.",
+  "license": "MIT",
+  "files": [
+    "README.md",
+    "dist",
+    "src",
+    "!*/__tests__"
+  ],
+  "copyright.owner": "IBM Corp.",
+  "publishConfig": {
+    "access": "public"
+  },
+  "peerDependencies": {
+    "@loopback/core": "^2.14.1",
+    "@loopback/rest": "^9.1.3"
+  },
+  "dependencies": {
+    "tslib": "^2.0.0",
+    "webapi-parser": "^0.5.0"
```

Comment:
Do you expect to use `webapi-parser` in the production code too? IIUC, this dependency is used in tests only, so maybe it should go to `devDependencies` instead?

Reply:
`webapi-parser` is the package that converts the OpenAPI 3.0 spec into RAML. This package is a wrapper that triggers the "converter" when the OpenAPI 3.0 spec binding is updated.